specs/interop/supervisor.md

```diff
@@ -159,6 +161,27 @@ Describes a block.
 - `canonical`: `Hash` - output root at the latest canonical block
 - `pending`: `Bytes` - output root preimage
 
+#### `SupervisorSyncStatus`
+
+Describes the sync status of the Supervisor component.
+
+`OBJECT`:
+- `minSyncedL1`: `BlockRef` - block ref to the synced L1 block
+- `safeTimestamp`: `Int` - safe timestamp
+- `finalizedTimestamp`: `Int` - finalized timestamp
+- `chains`: `ARRAY` of `SupervisorChainSyncStatus` - List of L2 chains and their sync status
```

Comment:
This looks incorrect to me. It's object with `ChainID` keys and `SupervisorChainSyncStatus` values